.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests with node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+    concurrency: build
+    runs-on: ubuntu-latest
+    env:
+      CTP_PROJECT_KEY: ctp-subscriptions-ci-it
+      CTP_CLIENT_ID: ${{ secrets.CTP_CLIENT_ID }}
+      CTP_CLIENT_SECRET: ${{ secrets.CTP_CLIENT_SECRET }}
+      EXISTING_ORDER_TYPE_KEY: checkout-order
+      EXISTING_ORDER_LINE_ITEM_TYPE_KEY: checkout-order-line-item
```

Comment:
You don't need to set these 2 vars. They're just for running the setup script.

Reply:
https://github.com/commercetools/commercetools-subscriptions/pull/3/commits/f3f94ad65fbff6a4d89a0ce1012864e29d11d95c